src/webfaf/reports.py

```diff
@@ -1014,8 +1015,8 @@ def attach():
 
 def get_avg_count(first, last, count):
     diff = last - first
-    r_d = diff.days / 30.4  # avg month size
+    r_d = diff.days // 30.4  # avg month size
     if r_d < 1:
         r_d = 1
 
-    return int(round(count / r_d))
+    return int(round(count // r_d))
```

Comment:
Just / so the round() can work

Reply:
You are right, it should stay ```/``` because the second operator is ```float``` as the result of the previous division operation (or integer value of 1 when the final result is ```count``` value).